home-assistant-voice.yaml

```diff
@@ -258,6 +258,13 @@ switch:
     entity_category: config
     restore_mode: ALWAYS_OFF
     internal: true
+  - platform: gpio
+    pin: GPIO46
+    id: grove_port_power
+    name: "Grove port power"
+    disabled_by_default: true
```

Comment:
I don’t want to expose this as an entity at all. People need to update yaml. Maybe we can make it a substitute value at the top of the file that people seee when they adopt it ?